.changeset/om-skip-context-messages.md

```
@@ -0,0 +1,18 @@
+---
+'@mastra/memory': patch
+---
+
+Fixed Observational Memory saving ephemeral `context` messages to storage as durable user messages.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)